internal/policy/policy.go

```diff
@@ -152,6 +152,27 @@ func LoadCurrentState(ctx context.Context, repo *git.Repository) (*State, error)
 	return LoadState(ctx, repo, latestEntry)
 }
 
+// LoadFirstState returns the State corresponding to the repository's first
+// active policy. It does not verify the root of trust since it is the initial policy.
+func LoadFirstState(ctx context.Context, repo *git.Repository) (*State, error) {
+	latestEntry, _, err := rsl.GetLatestReferenceEntryForRef(repo, PolicyRef)
+	if err != nil {
+		return nil, err
+	}
+
+	firstEntry, _, err := rsl.GetFirstEntry(repo)
+	if err != nil {
+		return nil, err
+	}
+
+	allPolicyEntries, _, err := rsl.GetReferenceEntriesInRangeForRef(repo, firstEntry.ID, latestEntry.ID, PolicyRef)
```

Comment:
we're really doing two passes at walking almost all the way back the RSL. If #344 lands first, this can just use rsl.GetFirstEntryForRef, right?

Reply:
Will fix once #344 lands

Reply:
Fixed